README.md

```diff
@@ -5,6 +5,26 @@ repository at <https://git.kitenet.net/index.cgi/git-annex.git/>.  The remote
 The local `master` branch in this repository is only used to store CI
 configuration and associated scripts.
 
+## Submitting Patches
+
+Patches to the git-annex source code can be added by opening a pull request
+against `master` that adds a Git patch file with a `.patch` file extension to
+the `patches/` directory; the CI will then build git-annex with this patch (and
+any others currently in `patches/`) applied.  If the patch fails to apply or
```

Comment:
any ordering is enforced (e.g. via `sort` and thus prefix with some `001` etc advised) among the sequence of patches?

Reply:
The second sentence of the third paragraph states "Patches in `patches/` are applied in lexicographic filename order."  Is that what you were asking?

Reply:
yes, thanks, missed that